extensions/functions_comparison.yaml

```diff
@@ -214,3 +214,30 @@ scalar_functions:
         variadic:
           min: 2
         return: any1
+  -
+    name: "least"
+    description: >-
+      Returns the smallest value in `List<T>`. Only return null if 'all' arguments evaluate to null.
+
+      String comparison is done in lexicographical ordering, one character at a time, from left to right.
+      Uppercase letters are less than lowercase letters.
+    impls:
+      - args:
+          - value: List<T>
```

Comment:
I don't think the value should be a List<T>. I think it should just be a T. Variadic means that we can execute something like: 

```least(4,6,8,10).```

I don't understand what something like the below would mean. This is what you are currently indicating with the arg type of  `List<T>`, min:2.

```
least([2,4],[5,8])
```

Reply:
Updated! Thanks!